css/index.css

```diff
@@ -101,7 +132,7 @@ p {
     justify-content: space-evenly;
 }
 
-.middle-content .boxes .box {
+.middle-content .boxes #box1, #box2, #box3, #box4, #box5, #box6, #box7, #box8, #box9, #box10 {
```

Comment:
Since .boxes is the parent of #box1, #box2 etc.  It will automatically inherit the style of boxes unless you override the CSS styles.